src/main/java/opendota/Parse.java

```diff
@@ -146,7 +151,15 @@ public UnknownItemFoundException(String message) {
 	private GreevilsGreedVisitor greevilsGreedVisitor;
 	private TrackVisitor trackVisitor;
     private ArrayList<Boolean> isPlayerStartingItemsWritten;
-    
+
+    //Draft stage variable
+    Integer draftStage = 0;
+    int[] draftHeroes = new int[22];
```

Comment:
I think you can move this down into the onTick function since there's no reason for it to be global